### DISCUSSION TITLE
how to test TPC-H throughput in Gluten?

### DISCUSSION THREAD

**Discussion by wanweiqiangintel** [apache/gluten#3562] [Q&A]
I want to do TPC-H throughput test with Gluten, but don't know how to do it in the standard way.

**Comment by zhouyuan** [comment]
@wanweiqiangintel 
the general idea for throughout tests is 
- start the thrift server: https://spark.apache.org/docs/latest/sql-distributed-sql-engine.html
- submit query's with beeline client
- increase the load by submitting queries with multiple beeline client

thanks, -yuan

**Reply by wanweiqiangintel** [reply]
When I create an external database with Hive, I got the error as below: 
![image](https://github.com/oap-project/gluten/assets/89506884/f579f88f-52d7-4d92-9ce7-adf6a56572aa)

Do you have any suggestions? Thanks!

**Reply by zhouyuan** [reply]
it looks like your gluten/velox is compiled w/o hdfs. 

-yuan
